docs/reference/design/actions.md

```diff
@@ -180,10 +179,10 @@ It is common practice (especially in RESTful APIs) to be able to accept incoming
 
 In other words: Praxis will inject a `:reference` parameter to the payload, pointing to the defined default MediaType of the endpoint (refer to [MediaType](media-types) for more info). It is for this reason that the following `create` action payload definition is enough if the default MediaType of the corresponding `Post` resource has those same attribute names defined.
 
-```
+```ruby
 action :create do
   routing { post '' }
-  payload do
+  payload reference: MediaTypes::Post do
```

Comment:
I think we need this here, right?

Reply:
I think the default mt is used as a reference if there is one...but I need to double check

Reply:
I've confirmed that's the case. I.e., if there is not reference option in the payload, Praxis will automatically add the default mediatype as a reference (if there is a default mediatype defined). 

So, we should delete the reference in the docs you added.